### DISCUSSION TITLE
Translated Relationship in filament version 3

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#8959] [Help] [answered]
i have this relationship but the result is always coming like the attached image, does filament provide a method to customize over the returned data from relation 

```
     Forms\Components\Select::make('users')
                            ->label(__('Test'))
                            ->multiple()
                            ->relationship('users', 'job')

```

![Screenshot from 2023-10-08 16-38-43](https://github.com/filamentphp/filament/assets/56083879/53531cfc-d423-45c2-a33f-971d12591e8a)

**Comment by Hegabovic** [comment]
@leandrocfe can you support here :D

**Reply by leandrocfe** [reply]
```php
    ->getOptionLabelsFromRecordUsing(fn ($record) => $record->job)
```
?

**Reply by Hegabovic** [reply, marked as answer]
```
->getOptionLabelFromRecordUsing(fn ($record) => $record->job)
```
it is working :D yet again thanks for your help

**Reply by leandrocfe** [reply]
mark as answered please
